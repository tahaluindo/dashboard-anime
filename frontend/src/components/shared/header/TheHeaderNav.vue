<template>
  <nav class="flex justify-between gap-1">
    <slot />
  </nav>
</template>

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({});
</script>

<style lang="postcss">
</style>