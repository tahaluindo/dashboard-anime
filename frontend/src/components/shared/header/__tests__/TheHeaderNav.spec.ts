import { mount } from "@vue/test-utils";
import TheHeaderNav from "../TheHeaderNav.vue";

describe("TheHeaderNav component test", () => {
  it("should render the default component", () => {
    const wrapper = mount(TheHeaderNav, {
      shallow: true,
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
