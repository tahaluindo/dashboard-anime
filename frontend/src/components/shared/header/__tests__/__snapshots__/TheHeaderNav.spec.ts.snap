// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheHeaderNav component test should render the default component 1`] = `"<nav class=\\"flex justify-between gap-1\\"></nav>"`;
